Adyen.EcommLibrary/ApiKeyAuthenticatedService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Adyen.EcommLibrary.Service;
+
+namespace Adyen.EcommLibrary
+{
+    public class ApiKeyAuthenticatedService : AbstractService
+    {
+        protected new bool RequiresApiKey { get;  set; } = true;
+
+        protected ApiKeyAuthenticatedService(Client client) : base(client)
```

Comment:
do we need this method ?

Reply:
The usage is to  distinguish  the cases need basic authentication or authentication via xapi key
``` 
      if (config.RequiresXApiKey)
      {
         AddHeaders(config, httpWebRequest);
      }
      else
      {
         CreateBasicAuthentication(config, httpWebRequest);
      }
```
 We can talk about it.